fedora-pr-review-guide.md

```diff
@@ -74,6 +76,17 @@ Copy & paste to your PR review.
     - Set it to ASSIGNED (and *take* it) when you start working on it
     - Put the link to the PR in the BZ once it exists (even WIP), set the status to POST
 
+### Patches
+
+- If your PR adds new patches (or significantly reworks some patches):
+  - Properly document the purpose of the patch in the specfile.
+    Detailed commit messages in the patches are good,
+    but a person who will read the spec should not need to open the patch to figure out what it is for.
+  - If you backport patches (e.g. from upstream)
+    - Make sure the authorship and/or origin of the patch is well traceable.
+      - Don't replace/strip the original git author information from a backported commit,
+        use a git-formatted patch with commit message and author.
```

Comment:
How about an example how to create such patch if the original author was eg. `John Doe <john@doe.com>`?

Reply:
Added in fixup.